src/schema/v2/home/home_page_artist_interface.ts

```diff
@@ -0,0 +1,28 @@
+import { GraphQLInterfaceType, GraphQLString } from "graphql"
```

Comment:
This is pretty trivial, but can we do camelCase file names? No for things to not be 1:1, especially in our MPv2 world.

Reply:
Yup, no problem 👍 I based it on its siblings but didn’t notice it not match the rest.